src/steamship/agents/tools/video_veneration/did_video_generator_tool.py

```diff
@@ -0,0 +1,57 @@
+"""Tool for generating images."""
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+from steamship.agents.tools import ImageGeneratorTool
+from steamship.utils.repl import ToolREPL
+
+
+class DIDVideoGeneratorOptions(BaseModel):
+    source_url: str
+    """The URL of the source image to be animated."""
+
+    stitch: str
+
+    provider: Optional[str]
```

Comment:
Ditto here re being unsure (`provider`)

Reply:
Going to add the BaseModel's I've designed for them.